app/controllers/departments_controller.rb

```diff
@@ -1,6 +1,16 @@
 class DepartmentsController < ApplicationController
   load_and_authorize_resource
 
+  def index
+    if !params[:checked].nil? && current_employee.role != 'Admin'
```

Comment:
checked is not so useful name, rename to something better
use 'Admin' as constant (I think it is already defined)